src/test/scala/nl.knaw.dans.easy.deposit/StagedFilesTargetSpec.scala

```diff
@@ -70,8 +69,13 @@ class StagedFilesTargetSpec extends TestSupportFixture {
   it should "not replace a fetch file" in {
     (stagedDir / "some.thing").createFile().write("new content")
     val url = new URL("https://raw.githubusercontent.com/DANS-KNAW/easy-deposit-api/master/README.md")
-    assume(DDM.triedSchema.isAvailable)
-    val bag = newEmptyBag.addFetchItem(url, Paths.get("path/to/some.thing")).getOrRecover(e => fail(e))
+    val bag = newEmptyBag.addFetchItem(
+      url,
+      Paths.get("path/to/some.thing"),
+    ).getOrRecover { e =>
+      assume(!e.isInstanceOf[UnknownHostException])
```

Comment:
To allow building of `dans-bag-lib` it should also apply conditional ignores in `SaveSpec` and `FetchItemSpec`

Reply:
feel free to do a PR if needed 😉

Reply:
No immediate need, it does `assumeCanConnect(url)`. Though this new `getOrRecover` will be faster if there is a connection.
See https://github.com/DANS-KNAW/dans-bag-lib/blob/8a3e78451e63881441efac56e880293fef764cda/src/test/scala/nl/knaw/dans/bag/fixtures/CanConnect.scala#L34-L48